pytorch_lightning/utilities/collate_deepspeed_checkpoint.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# Modified script from https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/utils/zero_to_fp32.py
```

Comment:
do you think it is enought?

Reply:
Very good point, as a gist yes, but in our repo no :) let me update the licence!

Reply:
I added the original MS licence! (MIT)